Jetty 12.0.x 12191 debug listener (#12231)
eclipse/jetty.project: documentation/jetty/modules/operations-guide/pages/deploy/index.adoc

[[env-context-xml]]
== Deploying Environment Specific Context XML Files

xref:xml/index.adoc[Jetty context XML file] files can be applied to _all_ webapps deployed to a particular `environment`.

Add a properties file to the deployment directory that contains a property naming the location of the xml file to apply.
The properties file name must be prefixed by the name of the `environment`.

> For example `ee8.properties, ee8-more.properties ee8-other.properties` would all apply to the `ee8` environment.

Reviewer: ```suggestion
For example `ee8.properties, ee8-more.properties ee8-other.properties` would all apply to all contexts deployed in the `ee8` environment.
```

> The properties files contain at minimum one property whose name is prefixed with `org.eclipse.jetty.deploy.environmentXml`.

Reviewer: ```suggestion
If the property file(s) contain one or more properties whose names are prefixed with `org.eclipse.jetty.deploy.environmentXml`, then their values are used as additional context XML files to apply to all contexts deployed in the corresponding environment.
```

So for example `org.eclipse.jetty.deploy.environmentXml, org.eclipse.jetty.deploy.environmentXml.more, org.eclipse.jetty.deploy.environmentXml.other` are all acceptable as names.

> Each property configures the location of a context xml file to apply to a context when it is being created and deployed.

Reviewer: ```suggestion
Each property configures the location of a context XML file to apply to a context when it is being created and deployed.
```
